docs/manage/fleet/_index.md

```diff
@@ -3,51 +3,53 @@ title: "Fleet Management"
 linkTitle: "Fleet Management"
 weight: 30
 type: "docs"
-description: "Configure, control, debug, and manage your robots from the cloud at app.viam.com."
+description: "Configure, control, debug, and manage your robots from the cloud at app.viam.com on your own or with a team."
 tags: ["fleet management", "cloud", "app"]
+no_list: true
 aliases:
     - "/manage/fleet-management"
     - "/manage/app-usage"
     - "/product-overviews/fleet-management/"
 ---
 
-The [Viam app](https://app.viam.com) provides fleet management allowing you to:
+The [Viam app](https://app.viam.com) provides fleet management allowing you to work alone or in collaboration with others on one or more robots.
 
-- manage your robots and access to them with [locations](#locations), [organizations](#organization), and [permissions](#permissions)
-- [configure](#configuration) individual or groups of robots
-- [deploy code and machine learning models](#package-deployment) to robots
-- [remotely control](#remote-control) and [debug](#logs) robots
+## Work with groups of robots
 
-All communication happens securely over HTTPS using secret tokens that are in a robot's configuration.
+With Viam, you can organize {{< glossary_tooltip term_id="robot" text="robots" >}} into {{< glossary_tooltip term_id="location" text="locations" >}} and {{< glossary_tooltip term_id="organization" text="organizations" >}}.
 
-## Robots
+For example, you may have separate organizations for your robots at home and at work:
 
-An organizational concept, consisting of either one _{{< glossary_tooltip term_id="part" text="part" >}}_, or multiple _parts_ working closely together to complete tasks.
+{{<gif webm_src="../img/organizations.webm" mp4_src="../img/organizations.mp4" alt="An organization for personal robots and one for work robots.">}}
 
-For more information on robots and how to manage them, see [Robots](robots).
+Inside an organization, you can organize robots into one or more locations:
 
-## Locations
+![An image of two locations, New York, and Chicago, in one organization, Good Robots](../img/locations.png)
```

Comment:
The gif is much much much smaller than the image on large screens, which looks odd. Maybe the image should be smaller, or the gif should be bigger, or if the gif can't be bigger because it's low resolution, maybe just replace it with a higher resolution image? As cute as the animation is it doesn't add all that much compared to an equivalent image :)